controller/helper/process-survey-instances.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+const moment = require('moment');
+/**
+ * @module controller/helper/process-survey-instances
+ */
+
+/**
+ * Takes in a Survey Instances and processes it to get Complience chart details
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} Complience chart data
+ */
+function processSurveyInstances (surveys) {
+    return {
+        labels: pickupSubmissionTime(surveys),
+        datasets: pickTimeLeft(surveys)
+    };
+}
+
+/**
+ * Takes in a Survey Instances and get the dates to be shown on Complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} list of dates labels
+ */
+function pickupSubmissionTime (surveys) {
+    const dates = [];
+
+    for (let index = 0; index < surveys.length; index += 1) {
+        // dates[index] = formatDate(surveys[index].startTime);
+        dates[index] = moment(surveys[index].startTime).utc().format('MM/DD/YYYY HH:mm');
+    }
+    return dates;
+}
+
+/**
+ * Takes in a Survey Instances and get the % time left to be shown on complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} processed list of % time left data
+ */
+function pickTimeLeft (surveys) {
+    const percentages = [];
+
+    for (let index = 0; index < surveys.length; index += 1) {
+        percentages[index] = calculateTimeLeft(moment(surveys[index].startTime).utc(),
+                                                moment(surveys[index].endTime).utc(),
+                                                moment(surveys[index].actualSubmissionTime).utc());
+    }
+    const calData = [{label: '% Time left until daily survey expired',
+                    data: percentages}];
+
+    return calData;
+}
+
+/**
+* Takes in a Survey Instances and get the % time left to be shown on complience chart
```

Comment:
:art: please add a space in the lines after the open of a doc comment, so the stars line up

``` js
/**
 *
 *
 */
```